bdk-ffi/Cargo.toml

```diff
@@ -18,11 +18,11 @@ path = "uniffi-bindgen.rs"
 default = ["uniffi/cli"]
 
 [dependencies]
-bdk_wallet = { version = "1.0.0-beta.4", features = ["all-keys", "keys-bip39", "rusqlite"] }
-bdk_core = { version = "0.2.0" }
-bdk_esplora = { version = "0.18.0", default-features = false, features = ["std", "blocking", "blocking-https-rustls"] }
-bdk_electrum = { version = "0.18.0", default-features = false, features = ["use-rustls-ring"] }
-bdk_bitcoind_rpc = { version = "0.15.0" }
+bdk_wallet = { version = "1.0.0-beta.5", features = ["all-keys", "keys-bip39", "rusqlite"] }
```

Comment:
Here I think we should pin the bdk_wallet dependency to beta 5. The default cargo behaviour of updating to the latest patch broke my (and all of our) builds as of yesterday when they released beta 5 which contained a breaking change. This is not a bdk problem but really a cargo problem.

Reply:
I pushed an update to pin, but just double check its what you were asking for and I won't merge or anything until you give it the go ahead that this is good-